src/App.tsx

```diff
@@ -1,61 +1,98 @@
-import React from 'react';
-import 'bulma/css/bulma.css';
-import './App.scss';
+import React, { useState } from "react";
+import "bulma/css/bulma.css";
+import "./App.scss";
+import cn from "classnames";
+import { SortType } from "./types/SortType";
+import { SortOptions } from "./types/SortOptions";
 
 export const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
+  "Dumplings",
+  "Carrot",
+  "Eggs",
+  "Ice cream",
+  "Apple",
+  "Bread",
+  "Fish",
+  "Honey",
+  "Jam",
+  "Garlic",
 ];
 
+function sortGoods(goods: string[], { query, reversed }: SortOptions) {
+  const goodsToSort: string[] = [...goods];
+
+  goodsToSort.sort((a, b) => {
```

Comment:
better to add the check
```suggestion
  if ( query !== SortType.default) {
  goodsToSort.sort((a, b) => {
```